### PR TITLE
Added login and register form

### PR DESCRIPTION
**Pull request by ahmedjasarevic**
Homepage navbar is now fixed. Login / Register / Logout are now aligned.
I have done register and login form using bootstrap.When you click register it will make new account.Login will login you to your account.

### CODE REVIEW

File: aMovieDatabase/core/templates/login.html [hunk 1]
```diff
@@ -1,24 +1,40 @@
...
-        </form>
-      </div>
-    </div>
-  </div>
-{% endblock %}
+<head>
+      <link rel="stylesheet" type="text/css" href="{%static 'style.css' %}">
+</head>
+<body>
+<div class="login-form">
```
Comment (mehmedkadric) [inline aMovieDatabase/core/templates/login.html:8]: Login and register content should be moved to accounts app.

**Review by mehmedkadric — CHANGES_REQUESTED** [review]

File: aMovieDatabase/core/templates/login.html [hunk 2]
```diff
@@ -8,7 +8,8 @@
 <div class="login-form">
         <form method="post" action="{% url 'login' %}" >
                {% csrf_token %}
-        <h2 class="text-center">Log in</h2>
+            <h2>Login</h2>
+		<p class="hint-text">Welcome back! Log in to access your account.</p>
```
Comment (mehmedkadric) [inline aMovieDatabase/core/templates/login.html:12]: In login form, when I enter a username and move cursor to another input field, username text color turns into gray.

File: aMovieDatabase/core/templates/login.html (hunk 1 above)
Comment (ahmedjasarevic) [reply aMovieDatabase/core/templates/login.html]: Login and register are now moved to accounts app

File: aMovieDatabase/core/templates/login.html (hunk 2 above)
Comment (ahmedjasarevic) [reply aMovieDatabase/core/templates/login.html]: Now when you move cursor it stays black

**Comment by ahmedjasarevic** [comment]
I have moved navbar from base.html to includes/navbar.html to make code clear . For login and register I have added toast messages, you can find them in messages.html. For succes login, you will see username displayed at navbar.

**Review by mehmedkadric — APPROVED** [review]
LGTM! 🚀

**Pull request merged**